resources/liquibase/001-initial-schema.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd" logicalFilePath="001-initial-schema.xml">
+  <changeSet id="001" author="guseyn">
+    <sql>
+      CREATE TABLE "user" (
+        id SERIAL PRIMARY KEY,
+        login VARCHAR(64) NOT NULL,
+        avatar_url VARCHAR(100) NOT NULL,
```

Comment:
@Guseyn 100 chars may not be enough for the URL

Reply:
@yegor256 i think to put here 2000, https://stackoverflow.com/questions/417142/what-is-the-maximum-length-of-a-url-in-different-browsers

Reply:
@yegor256 maybe use text instead of VARCHAR

Reply:
@Guseyn yes, works for me